lib/puppet/application/device.rb

```diff
@@ -77,10 +77,10 @@ def help
 
 USAGE
 -----
-  puppet device [-d|--debug] [--detailed-exitcodes] [-V|--version]
+  puppet device [-d|--debug] [--detailed-exitcodes] [--deviceconfig <file>]
```

Comment:
Ah, nevermind, what you have is fine. For some reason, I thought it just said `[--deviceconfig]`